internal/app/networkd/pkg/reg/reg_test.go

```diff
@@ -118,7 +118,7 @@ func (suite *NetworkdSuite) TestHealthAPI() {
 	nClient := healthapi.NewHealthClient(conn)
 	hcResp, err := nClient.Check(context.Background(), &empty.Empty{})
 	suite.Assert().NoError(err)
-	suite.Assert().Equal(healthapi.HealthCheck_SERVING, hcResp.Messages[0].Status)
+	suite.Assert().Equal(healthapi.HealthCheck_NOT_SERVING, hcResp.Messages[0].Status)
```

Comment:
Feels janky setting the tests to this, but we don't have a good way to ensure that there is a valid arp entry during tests. So we'll just leave it here to verify basic ( netlink ) functionality.